### DISCUSSION TITLE
Cannot pin projects

### DISCUSSION THREAD

**Discussion by nlopez39** [community/community#84675] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, I am unable to pin projects onto my profile. The save pins button is grayed out. Please help!

**Comment by darkotodoric** [comment]
Something broke on the GitHub side, probably they will fix this soon. Until that, click on "Customize your pins", press f12 on the page and enter this JS in the console:

```
document.querySelector('.js-pinned-items-submit').disabled = false
```
